src/Integration.Vsix.UnitTests/Integration.Vsix.UnitTests.csproj

```diff
@@ -93,10 +93,6 @@
       <HintPath>..\..\packages\Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime.14.3.25407\lib\Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.VisualStudio.Language.Intellisense, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.VisualStudio.Language.Intellisense.14.0.23205\lib\net45\Microsoft.VisualStudio.Language.Intellisense.dll</HintPath>
```

Comment:
I deleted all of the _QuickInfoSource_ code and references, which included this reference.
The problem with the previous implementation was that when it was trying to find the issue message to display in the tooltip, it was using the current line number in the document. If the document is dirty (i.e. has unsaved edits), then the line numbers in the unsaved document wouldn't match with the line numbers in the saved version -> issues not found, or out of range problems if lines had been added to the live document.

The _tag_ mechanism already handles this problem - when a document is edited (i.e. text buffer is updated), the text spans in the "old" text buffer are mapped across to the corresponding text in the "new" text buffer. Since there is also support for providing tooltip information with _IErrorTag_, this means we can rely on the tagging framework code to show the correct tooltip text even when the document is dirty.